frontend/packages/core/src/stepper.tsx

```diff
@@ -191,11 +192,22 @@ const Step: React.FC<StepProps> = ({ children }) => <>{children}</>;
 export interface StepperProps extends Pick<MuiStepperProps, "orientation"> {
   activeStep: number;
   children?: React.ReactElement<StepProps>[] | React.ReactElement<StepProps>;
+  nonLinear?: boolean;
```

Comment:
Let's remove this and just extend the `Pick` to take this field as well, I'd rather not recreate if we dont need to